Lib/ufo2ft/filters/transformations.py

```diff
@@ -125,4 +131,8 @@ def filter(self, glyph):
         for a in glyph.anchors:
             a.x, a.y = matrix.transformPoint((a.x, a.y))
 
+        glyph.width, glyph.height = justscale.transformPoint(
```

Comment:
I wish fontTools `Transform` class had a `transformVector` method in addition to `transformPoint`, like Skia's SkMatrix's mapPoint/mapVector that would ignore the translation part (vectors being invariant under translation). So that you would not need to create that `justscale` (which is more like a 2x2 matrix) transform.

Reply:
Just add!

Reply:
here https://github.com/fonttools/fonttools/pull/2386